molstar-extension/site/index.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
```

Comment:
Is this the built version of the site? Should we do this just in CI perhaps?

Reply:
Yes, sure